docs/application/native/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,294 @@
+         return;
+     }
+
+     if (terminated == true)
+         dlog_print(DLOG_INFO, LOG_TAG, "The component is terminated");
+     else
+         dlog_print(DLOG_INFO, LOG_TAG, "The component is not terminated");
+     ```
+
+   - Clone the given context handle:
+
+     ```
+     component_context_h component_context_cloned = NULL;
+
+     ret = component_context_clone(&component_context_cloned, component_context);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE) {
+         dlog_print(DLOG_ERROR, LOG_TAG, "Failed to clone component context. error(%d)", ret);
+         return;
+     }
+     ```
+
+   - Get the state of the component:
+
+     ```
+     component_state_e state;
+
+     ret = component_context_get_component_state(component_context, &state);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE) {
+         dlog_print(DLOG_ERROR, LOG_TAG, "Failed to get component state. error(%d)", ret);
+         return;
+     }
+
+     dlog_print(DLOG_INFO, LOG_TAG, "Component State: %d", state);
+     ```
+
+   - Check whether the component is running as a sub component of the group:
+
+     ```
+     bool is_subcomponent = false;
+     ret = component_context_is_subcomponent(component_context, &is_subcomponent);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE) {
+         dlog_print(DLOG_ERROR, LOG_TAG, "Failed to check whether the component is running as sub component of the group. error(%d)", ret);
+         return;
+     }
+     ```
+   - Resume the running component:
+
+     ```
+     ret = component_manager_resume_component(component_context);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE) {
+         dlog_print(DLOG_ERROR, LOG_TAG, "Failed to resume component. error(%d)", ret);
+         return;
+     }
+     ```
+     When the component is frame-component, it will appear on the screen.
+
+   - Terminate the running component in the background:
+
+     ```
+     ret = component_manager_terminate_bg_component(component_context);
+     if (ret != COMPONENT_MANAGER_ERROR_NONE) {
+         dlog_print(DLOG_ERROR, LOG_TAG, "Failed to terminate bg component. error(%d)", ret);
+         return;
+     }
+     ```
+
+3. When you no longer need the component context, call `component_context_destroy()` to remove the handle and release all resources to prevent memory leaks:
+
+   ```
+   ret = component_context_destroy(component_context);
+   if (ret != COMPONENT_MANAGER_ERROR_NONE) {
+       dlog_print(DLOG_ERROR, LOG_TAG, "Failed to destroy component context. error(%d)", ret);
+       return;
+   }
+   ```
+
+<a name="filter"></a>
+## Managing Component Information
+
+To get the installed information and its details, and to operate on the information:
+
+1. Get the information of the currently-installed component with `component_manager_get_component_info()`. Use the obtained component ID and `component_info_h` handle of the component with received information as the parameter.
+
```

Comment:
"To get a component's information, the component must be installed." is missing here.
Is it OK not to include that sentence?

Reply:
Done